azure/pipelines/templates/deploy.yml

```
@@ -243,6 +244,7 @@ jobs:
                 -dsiApiSecret "${{parameters.dsiApiSecret}}"
                 -ucasUsername "${{parameters.ucasUsername}}"
                 -ucasPassword "${{parameters.ucasPassword}}"
+                -ucasZipPassword "${{parameters.ucasZipPassword}}"
```

```suggestion
                -ucasZipPassword "$(ucasZipPassword)"
```

I think this is the new recommended way of doing this.

---

I'd prefer to do this in a different PR for everything to avoid inconsistency here as well.